src/main/java/org/ladocuploader/app/submission/conditions/AddDocumentsSelected.java

```diff
@@ -0,0 +1,12 @@
+package org.ladocuploader.app.submission.conditions;
+
+import formflow.library.data.Submission;
+import org.springframework.stereotype.Component;
+
+@Component
+public class AddDocumentsSelected extends BasicCondition {
+    @Override
+    public Boolean run(Submission submission) {
+        return run(submission, "addIncome", "true");
```

Comment:
The plan is to change the name of this class as well as the "addIncome" input name to something more related to this.